docs/synapse/userguides/storm_adv_methods.ipynb

```diff
@@ -974,6 +922,16 @@
     "# Close cortex because done\n",
     "_ = await core.fini()"
    ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    ".. _Types: https://synapse.docs.vertex.link/en/latest/synapse/autodocs/stormtypes_prims.html\n",
```

Comment:
These should be relative links so they point to the current documentation build and not the latest build.

Reply:
Darn, thought I caught all of these...